### ISSUE TITLE
Can switch to view request by name or url

### ISSUE THREAD

**Issue by TheBlackReverand** [usebruno/bruno#3181]
### I have checked the following:

- [X] I've searched existing issues and found nothing related to my issue.

### Describe the feature you want to add

Add possibility to view request by name (like today) or by url (like my preview).

![image](https://github.com/user-attachments/assets/d12a8e01-ddb6-4896-a0cf-fec4c862eaf7)

On version 1.30.1 I can make a test on edit the file `packages/bruno-app/src/components/Sidebar/Collections/Collection/CollectionItem/index.js`

```js
<span className="item-name" title={item.name}>
  {item.type === 'http-request' ?
      item.request.url.substring(item.request.url.indexOf('/', 8),
                                  (item.request.url.includes('?') ? item.request.url.indexOf('?') : item.request.url.length)
                                )
    : item.name
  }
</span>
```

I hardly replace the name of item 'http-request' by a live calculated path.
I exclude the first part ==> https://my-server
I exclude query parameters too.

We need a new param on "Collection / Preferences / Display" for make this choice.

We maybe need to review the path storage for make it great?

Store a base path (https://my-server) on bruno.json file (need to support variables for environment support)
Store the path of request without query on each request bru file
Expose BasePath + RequestPath + live calculated queryParams on query bar or for runner

I just expose an idea to talk about it :)

### Mockups or Images of the feature

![image](https://github.com/user-attachments/assets/3ed79e07-dcfc-4ca5-b4d6-23750a3def63)

**Comment by Tehhs** [comment]
I agree, having an option to display by url would be cool.

**Comment by LCLLL** [comment]
yes, this way directory searches can also be matched with request urls.

**Comment by wideweide** [comment]
> yes, this way directory searches can also be matched with request urls.

yes , I'm not care how to show, but filter with url and more info is required

**Comment by TheBlackReverand** [comment]
I take time to finish this.

Can switch in Preferences / Display / Request View
![image](https://github.com/user-attachments/assets/eaa9694d-e4ec-4e88-827e-e5b6a5ff7a6f)

Search bar work with URL to
